packages/mdc-top-app-bar/fixed/foundation.ts

```diff
@@ -21,39 +21,33 @@
  * THE SOFTWARE.
  */
 
+import {MDCTopAppBarAdapter} from '../adapter';
 import {cssClasses} from '../constants';
-import MDCTopAppBarAdapter from '../adapter';
-import MDCTopAppBarFoundation from '../foundation';
+import {MDCTopAppBarFoundation} from '../standard/foundation';
 
-/**
- * @extends {MDCTopAppBarFoundation<!MDCFixedTopAppBarFoundation>}
- * @final
- */
 class MDCFixedTopAppBarFoundation extends MDCTopAppBarFoundation {
   /**
-   * @param {!MDCTopAppBarAdapter} adapter
+   * State variable for the previous scroll iteration top app bar state
    */
-  constructor(adapter) {
+  private wasScrolled_ = false;
+
+  /* istanbul ignore next */
+  constructor(adapter?: Partial<MDCTopAppBarAdapter>) {
     super(adapter);
-    /** State variable for the previous scroll iteration top app bar state */
-    this.wasScrolled_ = false;
 
     this.scrollHandler_ = () => this.fixedScrollHandler_();
   }
 
   init() {
```

Comment:
can we just remove the init and destroy in this foundation?

Reply:
uhhh... yeah! derp